zerver/lib/zulip_update_announcements.py

```diff
@@ -146,7 +146,7 @@ def is_group_direct_message_sent_to_admins_within_days(realm: Realm, days: int)
         event_type=RealmAuditLog.REALM_PROPERTY_CHANGED,
         extra_data__contains={
             RealmAuditLog.OLD_VALUE: None,
-            RealmAuditLog.NEW_VALUE: 0,
+            RealmAuditLog.NEW_VALUE: realm.zulip_update_announcements_level,
```

Comment:
This change seems wrong -- probably we want to just skip asserting what value it changed from `None` to at all? Oh, I see, but you're checking for a change from `None` to the current value.

But I think this function is unsafe to call and will fail in the event that an organization disabled having a `zulip_update_announcements_stream` at some point in the past -- the current value for the realm could be a couple entries old but still not the first value it moved to from `None`.

So yeah I think we just skip the `NEW_VALUE` part here.